src/advanced-bloom/AdvancedBloomFilter.ts

```diff
@@ -1,10 +1,36 @@
-import { Filter, FilterSystem, GlProgram, GpuProgram, PointData, RenderSurface, Texture, TexturePool } from 'pixi.js';
+import {
+    deprecation,
+    Filter,
+    FilterSystem,
+    GlProgram,
+    GpuProgram,
+    PointData,
+    RenderSurface,
+    Texture,
+    TexturePool,
+    // eslint-disable-next-line camelcase
+    v8_0_0,
+} from 'pixi.js';
 import { vertex, wgslVertex } from '../defaults';
 import { KawaseBlurFilter } from '../kawase-blur/KawaseBlurFilter';
 import fragment from './advanced-bloom.frag';
 import source from './advanced-bloom.wgsl';
 import { ExtractBrightnessFilter } from './ExtractBrightnessFilter';
 
+type DeprecatedPixelSizeValue = PointData | number[] | number;
+
+interface DeprecatedAdvancedBloomFilterOptions
+{
+    threshold: number,
+    bloomScale: number,
+    brightness: number,
+    kernels: number[] | null,
+    blur: number,
+    quality: number,
+    pixelSize: DeprecatedPixelSizeValue,
+    resolution: number,
+}
+
```

Comment:
This is a pretty verbose way to deprecate `pixelSize` type. Another approach is to change the name. For instance:

```ts
export interface AdvancedBloomFilterOptions {
    size?: PointData,
    /** @deprecated use size instead */
    pixelSize?: PointData | number[] | number,
}
```
This way you don't duplicate options. With renaming you can do clean-breaks on the `pixelSize`, `pixelSizeX`, `pixelSizeY` setters/getters. We want to make sure that `@deprecated` doc tag is added so that it shows up in user's IDE. E.g., **filter.~~pixelSizeX~~ = 10**

If you don't want to rename, you can do something like this:

```ts
interface DeprecatedAdvancedBloomFilterOptions extends Omit<AdvancedBloomFilterOptions, 'pixelSize'> {
    /** @deprecated use PointData only */
    pixelSize?: PointData | number[] | number;
}
```
At least this way you don't duplicate the interface.